README.md

```diff
@@ -1,6 +1,102 @@
 # ShopifyImportMassager
 
-**TODO: Add description**
+The script takes Shopify CSV exports and converts the order name in the csv to
+the _real_ underlying order number. This makes it easy to import the shopify
+data even if the customer is using multiple different order prefixes and suffixes
+and having third-party mechanisms injecting orders as well.
+
+## Warning on what happens to the data
+**Warning**
+Because the order names are being changed, it may mean that finding a specific
+_historical_ order number might not work as planned.
+
+For example, an order might have an order name of `E-75123-1` that comes from
+an external (non-shopify-store source), which gets converted to an order number
+of `#78775` because of the underlying shopify order numbering.
+
+This order name gets converted in all the associated files (orders, transactions
+and returns). So the order naming is _consistent_ across the files when the
+shopify imports happen, so there's no issue with data getting properly linked.
+However, it does mean that the original order name of `E-75123-1` does not exist
+in the Solve data store. Generally this isn't so much an issue because we import
+historical data to get our summary data, not to look at individual orders.
+
+### Preparing to run the massager
```

Comment:
This was my initial thoughts of how I would make this more generic -- have a configuration file template that the user configures to the specific import. However, making and renaming a file copy can be tedious and prone to mistakes. When I got further into the code I realized that I could actually run the conversion/export process in a much easier way with a single command.

So I leave this here as an example of a change in thinking driven by more simplicity and user-friendliness.